templates/errors/session-expired.html

```diff
@@ -3,7 +3,7 @@
 {% set page_title = _("Session expired") %}
```

Comment:
Should this now be `Session timed out` then?

Reply:
Changed page title to `Session timed out` (and couple tests).